scripts/install/lidarr.sh

```diff
@@ -48,15 +51,14 @@ Type=simple
 User=${user}
 Group=${user}
 Environment="TMPDIR=/home/${user}/.tmp"
```

Comment:
not mutli user, is this needed?

Reply:
Yes, the service needs to run as the user as it will be managing into that user's download folder. Otherwise we can get into permission nightmares quite fast as users would have to be made aware of the clusterfuck.